portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -42,12 +42,54 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be deleted"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeDeleted {
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131223 CanBeDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
+
+		Form.closeSidebar();
+
+		Form.addTextField(fieldPositionNumber = "1");
+
+		Form.editElementSetFieldLabel(fieldValue = "Text");
+
+		Form.closeSidebar();
+
+		Click(
+			key_name = "Paragraph",
+			locator1 = "FormFields#FIELD_TARGET_LABEL");
+
+		Click(locator1 = "Icon#VERTICAL_ELLIPSIS_BUTTON");
+
+		Click(
+			key_selectOption = "Delete",
+			locator1 = "FormViewBuilder#SELECT_FIELD_OPTIONS_LIST");
```

Comment:
The macro `FormsAdminNavigator.gotoDeleteField` can also help you to delete the Paragraph field.
I've checked it and you could implement it like this:

```
FormsAdminNavigator.gotoDeleteField(
			fieldLabel = "Paragraph",
			fieldName = "Paragraph");
```

You can replace the lines 72-80 to this macro, as it will have same impact and will make the code cleaner.

Reply:
Changes done